### ISSUE TITLE
No info response should be nil instead of {}

### ISSUE THREAD

**Issue by expez** [clojure-emacs/cljs-tooling#21]
The info op, for cljs, now returns `{}` when nothing is found.  I think we should be consistent with the return value for the clojure info op, which returns `nil`.  Using `nil` allows us to use `when-let` and friends to better effect.

**Comment by bbatsov** [comment]
Sounds reasonable. @cichli will you look into this?

**Comment by cichli** [comment]
@expez do you have a more specific test case? The following test passes w/o any code changes:

```clojure
(testing "Returns nil for non-resolved syms"
    (is (= nil
           (info 'non-existent-var)
           (info 'non-existent-var 'non-existent-ns)
           (info 'non-existent-var 'cljs.core))))
```

**Comment by cichli** [comment]
Oh, never mind, the problem's in `cider-nrepl` itself. Fixing now :-).

**Comment by cichli** [comment]
Should be fixed by clojure-emacs/cider-nrepl@21ff15d5b6f177bd091ace4da42405ea4ef0b235.
